SrMoffat/pangaea
# Basic UI Application Layout

### What does this PR do?
- [x] Introduces the general layout of the app
- [x] Adds major UI components for the page + assets

### How can it be manually tested?
1. Clone the repo
`git clone https://github.com/SrMoffat/pangaea.git`
2. Install dependencies 
`yarn install` or `npm install`
3. Start the app
`yarn start` or `npm start`

You should see the general layout as follows
<img width="1440" alt="Screenshot 2020-11-23 at 16 08 59" src="https://user-images.githubusercontent.com/31407254/99965678-3dd32180-2da6-11eb-9fb3-dd14418e5fc0.png">
